### ISSUE TITLE
tvOS Support

### ISSUE THREAD

**Issue by gurhub** [kovpas/BOString#8]
Good library, thank you for your afford. There is no support for tvOS, probably it's just about the Podspec file settings.  Because NSAttributedstring available under tvos too. tvOS is good to have. By the way when I add with copy and paste, it works like charm. 

Best.

**Comment by kovpas** [comment]
Yes, I'd be happy to enable tvos and watchos support, but I've never had a chance to work with these platforms. I believe that the following lines should be added to the [podspec](https://github.com/kovpas/BOString/blob/master/BOString.podspec#L11):

```
  s.tvos.deployment_target  = '9.0'
  s.watchos.deployment_target  = '3.0'

...

  s.tvos.frameworks = 'Foundation', 'UIKit'
  s.watchos.frameworks = 'Foundation', 'UIKit'
```

I can't test if this would work. Could you please test and confirm if it helps?
